prow/config.yaml

```diff
@@ -14752,6 +14752,35 @@ periodics:
     - name: docker-graph
       emptyDir: {}
 
+- interval: 24h
+  name: ci-kubernetes-multicluster-ingress-test
+  agent: kubernetes
+  labels:
+    preset-service-account: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/bootstrap:v20180215-b2a89850e
+      args:
+      - --repo=GoogleCloudPlatform/k8s-multicluster-ingress=master
+      - --root=/go/src
+      - --timeout=50
+      env:
+      - name: DOCKER_IN_DOCKER_ENABLED
+        value: true
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
+      volumeMounts:
+      - name: docker-graph
+        mountPath: /docker-graph
+      resources:
+        requests:
+          cpu: 4
```

Comment:
4 CPU? seems like a lot.

Reply:
Seems to be the standard across all jobs

Reply:
it's used on larger builds mostly. no idea if this job needs it or not.

Reply:
Removed based on other jobs that use gcloud-in-go image